### PR TITLE
Move neoboard react code into a separate package

### PR DESCRIPTION
**Pull request by maheichyk**
Changes to monorepo and introduces `@nordeck/matrix-neoboard-react-sdk` package with neoboard components.

**:heavy_check_mark: Checklist**

- [x] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [ ] Added or updated documentation.
- [ ] Tests for new functionality and regression tests for bug fixes.
- [ ] Screenshots or videos attached (for UI changes).
- [x] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

File: matrix-neoboard-widget/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,6 @@ ENV REACT_APP_VERSION=${REACT_APP_VERSION}
 ENV REACT_APP_REVISION=${REACT_APP_REVISION}
 
 ADD build /usr/share/nginx/html/
-ADD LICENSE /usr/share/nginx/html/LICENSE.txt
```
Comment (weeman1337) [inline matrix-neoboard-widget/Dockerfile:9]: Shouldn't we keep the licence?

**Review by weeman1337 — COMMENTED** [review]
My main question would be: Why is `matrix-neoboard-widget` not placed under `packages`?

Other than that looks like good work! :+1:

File: matrix-neoboard-widget/Dockerfile (hunk 1 above)
Comment (maheichyk) [reply matrix-neoboard-widget/Dockerfile:9]: This line is not needed, LICENSE is copied by `generate-disclaimer` script.

**Comment by maheichyk** [comment]
> My main question would be: Why is `matrix-neoboard-widget` not placed under `packages`?

It is done in the same way as in `matrix-widget-toolkit` and `meetings-widget`, so we put an app (or a widget) at the root level.

**Review by weeman1337 — APPROVED** [review]

**Review by weeman1337 — DISMISSED** [review]

**Review by weeman1337 — APPROVED** [review]

**Pull request merged**